yellowbrick/cluster/elbow.py

```diff
@@ -355,8 +356,8 @@ def draw(self):
         """
         # Plot the silhouette score against k
         self.ax.plot(self.k_values_, self.k_scores_, marker="D")
-        if self.locate_elbow and self.elbow_value_ is not None:
-            elbow_label = "$elbow\ at\ k={}, score={:0.3f}$".format(
```

Comment:
I think these slashes were originally there to ensure the latex math between the `$` rendered correctly, thanks for cleaning it up!